include/verilatedos.h

```diff
@@ -139,20 +139,7 @@
 # define VL_PREFETCH_RW(p)  ///< Prefetch pointer argument with read/write intent
 #endif
 
-#if defined(VL_THREADED) && !defined(VL_CPPCHECK)
-# if defined(_MSC_VER) && _MSC_VER >= 1900
-#  define VL_THREAD_LOCAL thread_local
-# elif defined(__GNUC__)
-#  if (__cplusplus < 201103L)
-#   error "VL_THREADED/--threads support requires C++-11 or newer only; use newer compiler"
-#  endif
-# else
-#  error "Unsupported compiler for VL_THREADED: No thread-local declarator"
-# endif
-# define VL_THREAD_LOCAL thread_local  // "thread_local" when supported
-#else
-# define VL_THREAD_LOCAL  // "thread_local" when supported
-#endif
+#define VL_THREAD_LOCAL thread_local
```

Comment:
Should just inline VL_THREAD_LOCAL now

Reply:
Still define it in case user code used it, wrap it in an ifdef VL_NO_LEGACY.

Reply:
ifndef VL_NO_LEGACY that is